js/components/NewAppModalComponent.jsx

```diff
@@ -164,13 +164,15 @@ var NewAppModalComponent = React.createClass({
     var errors = this.state.errors;
 
     var generalErrors = errors.filter(function (e) {
-        return (e.attribute === "general");
-      });
+      return (e.attribute === "general");
+    });
 
     /* jshint trailing:false, quotmark:false, newcap:false */
     /* jscs:disable disallowTrailingWhitespace, validateQuoteMarks, maximumLineLength */
     var errorBlock = generalErrors.map(function (error, i) {
```

Comment:
Normally we will wrap html in return statements with a set of parenthesis, to make it more readable:
```javascript
var errorBlock = generalErrors.map(function (error, i) {
  return (
    <p key={i} className="text-danger">
      <strong>{error.message}</strong>
    </p>
  );
});
```